django/contrib/sites/management.py

```diff
@@ -2,13 +2,13 @@
 Creates the default Site object.
 """
 
-from django.apps import apps
+from django.apps import apps as global_apps
 from django.conf import settings
 from django.core.management.color import no_style
 from django.db import DEFAULT_DB_ALIAS, connections, router
 
 
-def create_default_site(app_config, verbosity=2, interactive=True, using=DEFAULT_DB_ALIAS, **kwargs):
+def create_default_site(app_config, verbosity=2, interactive=True, using=DEFAULT_DB_ALIAS, apps=global_apps, **kwargs):
```

Comment:
This doesn't have `app_config = apps.get_app_config(app_label)` as in the other changes -- is it intentional?